assets/js/theme/cart.js

```diff
@@ -125,30 +131,35 @@ export default class Cart extends PageManager {
         });
     }
 
-    cartEditOptions(itemId) {
+    cartEditOptions(itemId, productId) {
+        const context = { productForChangeId: productId, ...this.context };
         const modal = defaultModal();
         const options = {
             template: 'cart/modals/configure-product',
         };
 
         modal.open();
+        $('#modal').find('.modal-content').addClass('hide-content');
 
         utils.api.productAttributes.configureInCart(itemId, options, (err, response) => {
             modal.updateContent(response.content);
+            const $productOptionsContainer = $('[data-product-attributes-wrapper]', $('#modal'));
+            const modalBodyReservedHeight = $productOptionsContainer.outerHeight();
+            $productOptionsContainer.css('height', modalBodyReservedHeight);
+
+            this.productDetails = new CartItemDetails($('#modal'), context);
```

Comment:
in cartEditOptions we are catching modal several times by selector. Can we catch it once and then reuse?

Reply:
good point. thanks